base/exports.jl

```diff
@@ -1087,8 +1087,15 @@ eval(Expr(:public,
     :Generator,
     :ImmutableDict,
     :OneTo,
+    :TaggedString,
+    :TaggedChar,
     :UUID,
 
+# Tagged strings
+    :textproperties,
```

Comment:
> An error from the misc set??

`textproperties` is marked as public but not defined. It is causing the misc test failure.

Also, I don't think any of this should be part of the Base API (only the StyledStrings API), so Base should not mark any of these symbols as public. Symbol publicness, like export status, is on a per-binding basis, so it is different for Base to mark these as public than for StyledStrings to mark them as public.

Reply:
Ah oops, that should be `taggedstring`.

Regarding what's public, I was under the impression that with regard to `TaggedString`s we wanted to just make public API enough that they could be used. `annotate!`/`annotations` is so that the fields of the struct itself don't need to be directly accessed.

Reply:
Yes, you are right.